javers-persistence-sql/src/main/java/org/javers/repository/sql/SqlRepositoryBuilder.java

```diff
@@ -42,11 +47,20 @@ public SqlRepositoryBuilder withJSONConverter(JsonConverter jsonConverter) {
     public JaversSqlRepository build() {
         logger.info("starting up SQL repository module ...");
         bootContainer();
+
+        PolyJDBC polyJDBC = PolyJDBCBuilder.polyJDBC(dialectName.getPolyDialect())
+                .usingManagedConnections(new org.polyjdbc.core.transaction.ConnectionProvider() {
```

Comment:
Tak musi byc bo to ConnectionProvider z PolyJDBC a w JaversBuilder podajemy naszego ConnectionProvidera